pandera/checks.py

```diff
@@ -1022,3 +1021,43 @@ def _str_length(series: pd.Series) -> pd.Series:
                 f"str_length({min_value}, {max_value})",
             ),
         )
+
+    @classmethod
+    @st.register_check_strategy(st.isin_strategy)
+    @register_check_statistics(["allowed_values"])
+    def mustbein(cls, allowed_values: Iterable, strip_white_space_series: bool = False, **kwargs) -> "Check":
+        """Ensure series contains all values of within a list. The series must contain all values from the list.
+
+        :param allowed_values: The set of values that must be present. May be any iterable.
+        :param strip_white_space_series: Whether to strip white space from the series (only applies if str).
```

Comment:
I think we should remove this option from this check... since pandera checks don't modify the original data (only pandera data coercion does), I think this will lead to some undesirable behavior:

```python
values = {"A", "B", "C"}

check = pa.Check.set_eq(values, strip_white_space_series=True)
series1 = pd.Series(["A", "B", "C"])
series2 = pd.Series(["   A ", "B", "C"])

check(series1)  # passes
check(series2)  # passes
```

Both of these cases pass, but the second case seems like an odd use case, i.e. why would one want `series2` to pass validation? Since pandera is not a data cleaning library (it's data parsing + validation), I think stripping whitespace should be the responsibility of the user.